Source/WarpX.H

```diff
@@ -216,6 +216,9 @@ public:
     //! perform field communications in single precision
     static int do_single_precision_comms;
 
+    //! used shared memory algorithm for charge deposition
+    static int do_shared_mem_charge_deposition;
```

Comment:
Shouldn't this be a `bool`? Unless there's a plan to have multiple values, rather than just on/off

Reply:
yes, those `int`s should all be bool in a C++ code :)